packages/mongoose-store/src/config.ts

```diff
@@ -0,0 +1,24 @@
+// TODO move to modules to decide what datastore they should use.
+export default {
```

Comment:
It's fine to have it but I think that we should pass schema objects to this module